ADAL/src/request/ADAuthenticationRequest.m

```diff
@@ -148,12 +148,12 @@ - (void)setExtraQueryParameters:(NSString *)queryParams
 - (void)setClaims:(NSString *)claims
 {
     CHECK_REQUEST_STARTED;
-    if ([_requestParams claims] == claims)
+    if (_claims == claims)
```

Comment:
should it use `isEqualToString:` instead?

Reply:
Probably. But it would be then inconsistent with all other property setters, so I'd prefer to leave this one as is.